dptb/tests/test_threecenter.py

```
@@ -484,6 +484,132 @@ def test_so2_gate_wigner_cache_and_hermitian():
+    torch.manual_seed(0)
+    irreps = o3.Irreps("8x0e+8x1o+4x2e+2x3o")
+    x = torch.randn(64, irreps.dim, dtype=torch.float64)
+    R = o3.rand_matrix().to(torch.float64)
+    D = irreps.D_from_matrix(R)
+
+    sln = SeperableLayerNorm(irreps, eps=5e-3, affine=True, per_l=True, dtype=torch.float64)
+    assert torch.allclose(sln(x @ D.T), sln(x) @ D.T, atol=1e-8)
+
+    # unaffine per-l norm sends each l to ~unit RMS independently
+    sln0 = SeperableLayerNorm(irreps, eps=1e-6, affine=False, per_l=True, dtype=torch.float64)
+    y = sln0(x); off = 0
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)